sys/net/network_layer/fib/fib.c

```diff
@@ -633,13 +658,19 @@ static void fib_print_adress(universal_address_container_t *entry)
                 printf("  ");
             }
         }
+#ifdef MODULE_NG_IPV6_ADDR
```

Comment:
`NG_FIB_ADDR_PRINT_LEN` is forced to be `UNIVERSAL_ADDRESS_SIZE * 2` when not compiling with `MODULE_NG_IPV6_ADDR`, therefore the loop condition would be `false` on loop entry. This is known at compile time. Do you think compilers are smart enough to remove such loops? In case not, I added this guard here